### PR TITLE
Feat all uppercase plugin

### PR DESCRIPTION
**Pull request by shaun-ak**
**[New Plugin] all uppercase plugin** 
- Plugin that gives a true verdict if all characters in a sentence/text is uppercase.

**Related Issues:** (optional)
- #515

### CODE REVIEW

File: package.json
Comment (VisargD) [inline package.json]: Hi! What is the exact error that you are facing without adding ts-node package? Ideally, this package should not be a part of main dependencies.

Comment (shaun-ak) [reply package.json]: Hey @VisargD , I was not able to reproduce the issue somehow, so removed the package-lock json file and ts-node.

Comment (VisargD) [reply package.json]: I think you removed the entire `package-lock.json` file. Can you please revert both `package.json` and `package-lock.json` to the state they are in on the main branch?

Comment (shaun-ak) [reply package.json]: @VisargD I reverted all the changes for package-lock and package.json. Please check 😁

**Comment by shaun-ak** [comment]
Hey @VisargD , any update on this?

I am waiting for this to get merged, I am looking to work on all other plugins once this is merged (and if any changes are required, I will know it from this PR and will not repeat it for other plugins)

**Comment by VisargD** [comment]
Hey @shaun-ak. Sorry for the delay. I had one last suggestion. Can we add this to the `default` plugin directory instead of creating a new plugin? This is a deterministic checks so it fits well with other default plugins.

**Comment by shaun-ak** [comment]
Sure, will do that. Thanks.

**Comment by shaun-ak** [comment]
Hey @VisargD , I moved the plugin to default, please re-review.

File: plugins/default/manifest.json
Comment (VisargD) [inline plugins/default/manifest.json:425]: Hey. Can you please follow the same structure as used by other plugins in this file?

**Comment by shaun-ak** [comment]
Hey @VisargD , fixed it, please re-review.

**Review by VisargD — APPROVED** [review]

**Pull request merged**
